shared/locales/en/feedback.json

```diff
@@ -0,0 +1,54 @@
+{
+    "smartling": {
+        "string_format" : "icu",
+        "translate_paths": [{
+            "path": "*/title",
+            "key": "{*}/title",
+            "instruction": "*/note"
+        }]
+    },
+    "brokenSiteLabel": {
+        "title": "Which website is broken?",
+        "note": "Input label for broken site URL"
```

Comment:
"We're asking feedback from users. Here they are reporting a website url that is not working correctly."